### ISSUE TITLE
Sudden server crash after connection with presenter and initiator lost

### ISSUE THREAD

**Issue by kuleszdl** [bigbluebutton/bigbluebutton#12268]
**Describe the bug**
I was running a meeting with 30 participants and was connected from two machines (one as presenter that also initiated the meeting, one for audio). After aprox 2.5 hours the internet connection (on ISP-side) crashed, so my both machines were disconnected from the server. However, following this disconnection, the whole BBB server became unresponsive (SSH access was possible but not joining the meeting)

**To Reproduce**
Running 2.3-dev on bionic with all updates applied. Fresh installation, initially installed yesterday via bbb-install.sh
I tried to reproduce it but didn't suceed.

**Expected behavior**
Users with broken connection get disconnected, eventually meeting is ended (if expected?) but not the whole service becoming unresponsive.

**Actual behavior**
Whole service became unresponsive

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Firefox
 - Version 78.10 ESR

**Additional context**
I didn't find anything useful in the logs but maybe need some pointers where to look.

**Comment by hostbbb** [comment]
where you joining meeting from the demo pages? or other front end like greenlight/moodle?
when you tried to create a new meeting what error did you get?

**Comment by kuleszdl** [comment]
I created the meeting via greenlight. When creating the meeting, there was no issue and as explained it worked well for over 2.5 hrs until the crash. From there, the whole service (greenlight) was unreachable and the meeting as also ended for all participants in there (I am still trying to get a more detailed description from them).

**Comment by hostbbb** [comment]
did all the other 30 participants get kicked out of the meeting... if they still were in the meeting and working good,  then i would look at the greenlight logs, maybe the container hung for some reason, or greenlight crashed.

If they all became non responsive then points to server or bbb since greenlight not required to keep the meeting going for them.

**Comment by kuleszdl** [comment]
Yes, I am trying to reach out to the users to get exactly this feedback and will check back later.

**Comment by kuleszdl** [comment]
Received the feedback: The page first was just "blue", then after reloading participants got the message, that the meeting has ended. Trying to access the greenlight url (without the session id) led to the page loading endlessly.
